dist/about.html

```diff
@@ -96,7 +96,7 @@ <h2 id="contact">Contact us <a href="http://twitter.com/tosdr" class="btn btn-bi
 				</p>
 				<h3>Requesting a service review</h3>
 				<p>
-				  Unfortunately, we do not have the contributor capacity to do on-demand reviews. However, volunteers can review terms of service using our tool at <a href="https://edit.tosdr.org">https://edit.tosdr.org</a>, so perhaps one of your readers would be willing to do that? You can also discuss it with other reviewers, at <a href="https://forum.tosdr.org">forum.tosdr.org</a>.
+				  Unfortunately, we do not have the contributor capacity to do on-demand reviews. However, volunteers can review terms of service using our tool at https://edit.tosdr.org, so perhaps one of your readers would be willing to do that? You can also discuss it with other reviewers, at https://forum.tosdr.org</a>.
```

Comment:
Deleted the <a> tags because they were over a link themselves lol.

Reply:
The tag has an enclosing </a> in the end